charts/athens-proxy/templates/service.yaml

```diff
@@ -18,9 +18,6 @@ spec:
     port: {{ .Values.service.servicePort }}
     targetPort: 3000
     protocol: TCP
-    {{- if eq .Values.service.type "NodePort" }}
```

Comment:
This is backwards breaking and the NodePort should stay configurable.
The port is not hardcoded but set by default, IF service type is "NodePort" (which by default is "ClusterIP").